OMG.Classroom/API/Controllers/StudentController.cs

```diff
@@ -0,0 +1,64 @@
+﻿using DataAccessLayer;
+using DataAccessLayer.Entities;
+using DataAccessLayer.Repositories.StudentRepositories;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class StudentController : ControllerBase
+    {
+        //To be changed to service in the future and to DTOs/Models/ViewModels in the Future
+        private readonly StudentRepository _studentRepository;
+        public StudentController(StudentRepository studentRepository)
+        {
+            _studentRepository = studentRepository;
+        }
+        [HttpGet]
+        public async Task<IEnumerable<Student>> AllStudents()
+        {
+            return await _studentRepository.ReadAll().ToListAsync();
+        }
+        [HttpGet]
+        public async Task<Student> GetStudent(string id)
+        {
+            Guid guid = Guid.Parse(id);
+            return await _studentRepository.ReadAsync(guid);
+        }
+        [HttpPost]
```

Comment:
Same issue, why everything is a POST request, how can differentiate deleting, updating and creating?